ya.bat

```diff
@@ -8,13 +8,13 @@ call :dbg Ya: %YA_BAT_REAL%
 call :find_python
 if ERRORLEVEL 1 exit /b 1
 call :dbg Python: %YA_BAT_PYTHON%
-call %YA_BAT_PYTHON% %YA_BAT_REAL% %*
+call %YA_BAT_PYTHON% "%YA_BAT_REAL%" %*

```

Comment:
Please make separate pull request with this change

Reply:
#829